src/ol_dbt/models/reporting/_reporting__models.yml

```
@@ -0,0 +1,49 @@
+---
+version: 2
+models:
+- name: program_summary_report
+  description: MITx program summary report
```

Does this include xPRO or is it just MITx? 

```suggestion
  description: Program Summary report
```

```suggestion
  description: Program Summary report. A count of all users, enrollments (audit and verified), certificates and program credentials aggregated for each program. 
```

---

Updated. This includes both MITx and xPro.